services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -794,24 +795,24 @@ public Builder addApproaches(String... approaches) {
     abstract Builder approaches(@Nullable String approaches);
 
     /**
-     * Optionally, set which input coordinates should be treated as via way points.
+     * Optionally, set which input coordinates should be treated as separate legs.
      * <p>
      * Most useful in combination with  steps=true and requests based on traces
```

Comment:
Does `traces` make sense here?  Reading this javadoc, I'm still not entirely sure what `separateLegs` means. 

> set which input coordinates should be treated as separate legs.

By adding coordinates to a Directions API request, you are creating new legs.  I think it may make sense here to say that `separateLegs` is directly influencing the guidance behavior when set.

Reply:
https://github.com/mapbox/mapbox-java/pull/962#issuecomment-462809409